src/Data/These.purs

```diff
@@ -10,6 +10,10 @@ import Data.Maybe (Maybe(..), isJust)
 import Data.Traversable (class Traversable, class Foldable, foldMap, foldl, foldr)
 import Data.Tuple (Tuple(..))
 
+-- | Data type isomorphic to `α ∨ β ∨ (α ∧ β)` or
+-- | `Either a (Either b (Tuple a b))`.
+-- |
+-- | Most type classes instances work on the value of the `b` type.
```

Comment:
I'm not sure this line is needed, as this is the same `Functor` behavior that applies to any type with multiple type arguments.

Reply:
I think this line should become the new precedent because its helpful for new learners. However, I don't believe strongly in this either. Thoughts?

Reply:
I would prefer not to include it -- it's more important that new learners are able to associate this structure with `Functor` (or `Bifunctor`). For new learners I would rather have a brief example of using `map` with a `These` value in the quick start so they can begin to recognize a very wide-ranging pattern.

Reply:
Good point. Fixed